index.md

```diff
@@ -1,28 +1,26 @@
 ---
-layout: page
-title: Extra Unix Shell Material
+layout: lesson
+root: .
 ---
 
 Software Carpentry's [Introduction to the Unix Shell](http://swcarpentry.github.io/shell-novice/) describes the history of the shell, basic commands, and some of the philosophy that makes it such a powerful tool.  This lesson aims to supplement the material in that lesson with short topical lessons on particular shell features that are useful to researchers.  
 
-> ## Prerequisites{.prereq}
+> ## Prerequisites
 >
 > All of this material assumes that learners have mastered
 > the basic Unix shell lesson.  
+{: .prereq}
 
-## Topics
-
-1. [Navigating man pages](01-man-pages.html)
-2. [Connecting to remote machines](02-ssh.html)
-3. [Transferring files](03-file-transfer.html)
-4. [File permissions](04-permissions.html)
-5. [Directory structure](05-directory-structure.html)
-6. [Job control](06-job-control.html)
-7. [Aliases and the bashrc](07-aliases.html)
-8. [Environment variables](08-environment-variables.html)
```

Comment:
As Trevor points out, nothing has been removed. 
This commit,  	f83c399, just puts all of the original content 
into the `_episodes` directory, and after that, the front page
content is automatically generated.
It currently looks like

```
Software Carpentry’s Introduction to the Unix Shell describes the
history of the shell, basic commands, and some of the philosophy
that makes it such a powerful tool. This lesson aims to supplement
the material in that lesson with short topical lessons on particular
shell features that are useful to researchers.

    Prerequisites

    All of this material assumes that learners have
    mastered the basic Unix shell lesson.

    Getting Ready

    A short paragraph describing what learners need
    to do before tackling this lesson.

Other Resources

    Reference
    Discussion
    Instructor’s Guide

 	Setup 			Download files used on the lesson.
00:00 	Manual Pages 	
00:10 	Working Remotely 	How do I use ‘ssh’ and ‘scp’ ?
00:20 	Transferring files 	
00:30 	Permissions 	
00:40 	Directory structure 	
00:45 	Job control 	
00:50 	Aliases and the .bashrc 	
00:50 	Shell Variables 	
01:00 	Finish
```
although I believe, from looking at current lessons, that the
`Other Resources` and `Getting Ready` bits, which I left in
so as not to alter the existing content, would go from that
front page, as they have become items in the Extras menu.